### ISSUE TITLE
Number of training iterations for WaveGAN and Pulse2Pulse

### ISSUE THREAD

**Issue by Elizabaith** [AI4HealthUOL/SSSD-ECG#24]
Hi, I noticed that your SSSD-ECG model is trained 100,000 times by default, while WaveGAN as well as Pulse2Pulse are trained 3,000 times, is the comparison model also trained 3,000 times in your paper? 
Because I found that the quality of the generated signal doesn't seem to be very good when trained only 3000 times.

**Comment by juanlopezcode** [comment]
Hi, yes, I found a good convergence of SSSD-ECG around 100,000 iterations (number of batches), I could have evaluated on a validation set to make model selection and be more optimal, however I didn't. 
The baselines were trained on 3000 epochs (number of training data times), wavegan showed best result at 2000 while pulse2pulse at 2500 for the authors specific dataset, but they trained on 3000, that is why I took the 3000. See reference here: https://www.medrxiv.org/content/10.1101/2021.04.27.21256189v3.full.pdf#page=4.90

Juan

**Comment by Elizabaith** [comment]
Thank you!
